recipes/awscli-local/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "awscli-local" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a6a34cc7dd6774194cf43e67a6b93006fb4b395ac0dd5c9245da1b365c8ef767
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - awscli
+    - localstack-client
+    - python
+
+test:
+  commands:
+    - aws help > /dev/null
```

Comment:
Is there a real test for this?

Reply:
@scopatz May be able to execute a more robust command after "localstack" package has been added to the forge, but having a hard time finding an import style test for this package. All docs defer to aws package which is already in the forge.

Reply:
@scopatz awslocal.exe is also not recognized

The issues page of awscli-local seems to have others with this same problem: https://github.com/localstack/awscli-local/issues/6
https://github.com/localstack/awscli-local/issues/11